ui/pages/confirm-add-suggested-nft/__snapshots__/confirm-add-suggested-nft.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ConfirmAddSuggestedNFT Component should match snapshot 1`] = `
-{
+Object {
```

Comment:
Unsure of why Object has been added here?

Reply:
hmm not sure but it caused the test to fail. I ran `yarn lint:fix` earlier which I believe added this

Reply:
This was interesting because it was added by `yarn lint:fix`. When I ran the test locally, the test wasn't failing with `Object`.